### PR TITLE
Fix fault tolerance example

### PR DESCRIPTION
**Pull request by edthorne**
For #107

### CODE REVIEW

File: microprofile/rest-client/pom.xml [hunk 1]
```diff
@@ -64,7 +64,6 @@
                     <plugin>
                         <groupId>fish.payara.maven.plugins</groupId>
                         <artifactId>payara-micro-maven-plugin</artifactId>
-                        <version>${payara.miro.maven.version}</version>
```
Comment (edthorne) [inline microprofile/rest-client/pom.xml]: While this is unrelated to the original issue, I considered this low risk enough to include here. This property is not set and the version is specified in the parent pom.

Comment (edthorne) [reply microprofile/rest-client/pom.xml]: On second thought, there's more than this issue in the rest client example. I've reverted the change.

**Comment by OndroMih** [comment]
The example app runs currently with Payara Micro plugin 1.0.4 and Payara Micro. There's no need for this fix, however, we'll consider accepting it to update the project.
